packages/core/package.json

```diff
@@ -3984,7 +3984,7 @@
     },
     "devDependencies": {
         "@aws-sdk/types": "^3.13.1",
-        "@aws-toolkits/telemetry": "^1.0.205",
```

Comment:
note to reviewers: 

I've updated telemetry and added a new event emission to the /dev logic as the cost was very low and it was just one line in the logic. Let me know if you'd rather split this.